primer-miso/src/Primer/Miso.hs

```diff
@@ -110,6 +110,7 @@ import Primer.Core (
   ),
   GVarName,
   GlobalName (baseName, qualifiedModule),
+  ID (..),
```

Comment:
Nice, this will be useful when I look into whether node IDs are stable (in order to use them for animation purposes).